core/src/com/etheller/warsmash/util/Vector4.java

```diff
@@ -519,10 +516,7 @@ public boolean epsilonEquals(final Vector4 other, final float epsilon) {
 		if (Math.abs(other.z - this.z) > epsilon) {
 			return false;
 		}
-		if (Math.abs(other.w - this.w) > epsilon) {
-			return false;
-		}
-		return true;
+		return !(Math.abs(other.w - this.w) > epsilon);
```

Comment:
(Similar case to the Quadtree comment.)

Reply:
Please check 60abd007b901f47ca1d9b8cf424f7608c6e7b8bc, where I refactored Vector4.java.